packages/next/lib/typescript/getTypeScriptIntent.ts

```diff
@@ -9,7 +11,9 @@ export async function getTypeScriptIntent(
   baseDir: string,
   pagesDir: string
 ): Promise<TypeScriptIntent | false> {
-  const tsConfigPath = path.join(baseDir, 'tsconfig.json')
+  const config = await loadConfig(PHASE_PRODUCTION_BUILD, baseDir)
+  const tsConfigName = config.typescript.tsconfigPath
```

Comment:
The next config object should be passed in as a parameter instead of being read from the filesystem again

Reply:
Whoops, didn't think of that, Fixing now!